packages/@sanity/default-login/src/LoginDialog.js

```diff
@@ -3,12 +3,23 @@ import React from 'react'
 import PropTypes from 'prop-types'
 import authenticationFetcher from 'part:@sanity/base/authentication-fetcher'
 import pluginConfig from 'config:@sanity/default-login'
-import FullscreenDialog from 'part:@sanity/components/dialogs/fullscreen'
 import LoginDialogContent from 'part:@sanity/base/login-dialog-content'
```

Comment:
Can we move all of these parts to the `legacyParts` file, too?

Reply:
Yes! 👍